docs/source/release/v6.5.0/indirect_geometry.rst

```diff
@@ -7,8 +7,13 @@ Indirect Geometry Changes
 
 New Features
 ------------
-- A new set of scripts has been produced to provide a method for processing OSIRIS data. :ref:`isis-powder-diffraction-osiris-ref`.
+- A new set of scripts has been produced to provide a method for processing OSIRIS data. :ref:` ISIS Powder Diffraction Scripts - OSIRIS <isis-powder-diffraction-osiris-ref>`.
```

Comment:
The formatting for this link doesn't work. I'm not sure why - you may need to try a few things like removing the space between the first ` and the word ISIS or perhaps having a whole sentence rather than just a link between two full stops?